README_zh-CN.md

```diff
@@ -0,0 +1,111 @@
+<div align="center">
+  <img src="resources/mmtrack-logo.png" width="600"/>
+</div>
+
+[![PyPI](https://img.shields.io/pypi/v/mmtrack)](https://pypi.org/project/mmtrack)
+[![docs](https://img.shields.io/badge/docs-latest-blue)](https://mmtracking.readthedocs.io/en/latest/)
+[![badge](https://github.com/open-mmlab/mmtracking/workflows/build/badge.svg)](https://github.com/open-mmlab/mmtracking/actions)
+[![codecov](https://codecov.io/gh/open-mmlab/mmtracking/branch/master/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmtracking)
+[![license](https://img.shields.io/github/license/open-mmlab/mmtracking.svg)](https://github.com/open-mmlab/mmtracking/blob/master/LICENSE)
+
+[English](/README.md) | 简体中文
+
+文档: https://mmtracking.readthedocs.io/
+
+## 简介
+
+MMTracking是一款基于 PyTorch 的视频目标感知开源工具箱，是 [OpenMMLab](http://openmmlab.org/) 项目的一部分。
```

Comment:
Remove the unnecessary space, like the two next to `PyTorch`. The same to the entire doc.